docs/eventing/getting-started.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Getting Started with Eventing"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 9
+type: "docs"
+---
+
+This getting started guide will walk you through all the steps involved in a typical Knative use case. You will learn how to create, send, and verify events in Knative.
+
+## Before you begin
+
+You need:
+
+- A Kubernetes cluster with [Knative Eventing installed](../install/README.md). 
+
+## Setting up Eventing subcomponents
+
+Before you start to send events, you need to create the components needed to transport the events. This guide walks you through creating each Eventing subcomponent.
+
+### Configuring namespace for eventing
+
+In most Knative tutorials, you will operate inside of a namespace. A namespace groups together subcomponents so that you can easily organize them.
+
+1. Create the namespace. This command creates a namespace called **event-example**:
+
+```sh
```

Comment:
Each code block and any text must be nested within/under the list element with 4 spaces otherwise it renders poorly on the site (and breaks/resets numbers in the lists). Note: Our Hugo doc build only likes spaces (not tabs).
```suggestion
    ```sh
```